shared_model/interfaces/utils/query_error_response_visitor.hpp

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2018 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_QUERY_ERROR_RESPONSE_VISITOR_HPP
+#define IROHA_QUERY_ERROR_RESPONSE_VISITOR_HPP
+
```

Comment:
I do not think interfaces directory is a good place for this. Same for the second class. Maybe just utils?

Reply:
> I do not think interfaces directory is a good place for this

Why? It works over the ifaces

> Also do we need such utils in other places besides GetTransactions test?

I think it can be useful as a general API (well, maybe just more generic)

Reply:
It works over interfaces but it's not an interface by itself. Also at the moment interface directory contains only interfaces and I think it's a good design so let's keep it.

Reply:
Ok, agree